### PR TITLE
[docs] Updating install instructions to add packages.

### PR DESCRIPTION
**Pull request by angrycub**
Added Tabs component from Learn (will need @jescalan to approve).  Added information about installing from package.

[Preview Link](https://deploy-preview-8534--nomad-website.netlify.app/docs/install)

### CODE REVIEW

File: website/pages/docs/install/index.mdx [hunk 1]
```diff
@@ -7,28 +7,223 @@ description: Learn how to install Nomad.
...
+
+</Tab>
+</Tabs>
+
+-> **TIP:** Now that you have added the HashiCorp repository, you can install
+[Consul](https://learn.hashicorp.com/consul) and
+[Vault](https://learn.hashicorp.com/vault) with the same command.
+
+</Tab>
+<Tab heading="Homebrew(macOS)">
```
Comment (jescalan) [inline website/pages/docs/install/index.mdx:149]: I might add a space before the parens here?

File: website/pages/docs/install/index.mdx [hunk 2]
```diff
@@ -7,28 +7,223 @@ description: Learn how to install Nomad.
...
+~> **NOTE:** Homebrew and the Nomad formula are **NOT** directly maintained by
+HashiCorp. The latest version of Nomad is always available by manual
+installation.
+
+~> **NOTE**: On macOS, machines without a Java environment installed will be
+prompted to install Java because of a [known issue—#7865][gh-7865]. The linked
+GitHub issue has workarounds for this issue.
+
+</Tab>
+<Tab heading="Chocolatey(Windows)">
```
Comment (jescalan) [inline website/pages/docs/install/index.mdx:168]: same here

Comment (angrycub) [inline website/pages/docs/install/index.mdx:168]:
```suggestion
<Tab heading="Chocolatey (Windows)">
```

File: website/pages/docs/install/index.mdx (hunk 1 above)
Comment (angrycub) [inline website/pages/docs/install/index.mdx:149]:
```suggestion
<Tab heading="Homebrew (macOS)">
```

File: website/pages/docs/install/index.mdx [hunk 3]
```diff
@@ -7,28 +7,223 @@ description: Learn how to install Nomad.
...
-Downloading a precompiled binary is easiest, and we provide downloads over
-TLS along with SHA-256 sums to verify the binary.
+<Tabs>
+<Tab heading="Manual installation">
 
-## Precompiled Binaries ((#precompiled-binaries))
+You can download a [precompiled binary](https://nomadproject.io/downloads/) and
+run it on your machine locally. You can also verify the binary using the
+available SHA-256 sums. After downloading Nomad, unzip the package. Make sure
+that the `nomad` binary is available on your `PATH`, before continuing with the
```
Comment (schmichael) [inline website/pages/docs/install/index.mdx:22]:
```suggestion
that the `nomad` binary is available on your `PATH` before continuing with the
```

File: website/pages/docs/install/index.mdx [hunk 4]
```diff
@@ -7,28 +7,223 @@ description: Learn how to install Nomad.
...
+your shell). In that file you will see a line that starts with `export PATH=`,
+followed by a colon-separated list of locations. Add the location of the Consul
+binary to that list and save the file. Then reload your shell's configuration
+with the command `source ~/.bashrc`, replacing `bash` with the name of your
+shell.
+
+-> **Tip (Windows):** Add a location to your path via the GUI by navigating to
+`Environment Variables` in your system settings, and looking for the variable
+called `PATH`. You will see a semicolon-separated list of locations. Add the
+Consul binary's location to that list and then launch a new console window.
```
Comment (schmichael) [inline website/pages/docs/install/index.mdx:49]:
```suggestion
Nomad binary's location to that list and then launch a new console window.
```

**Pull request merged**